flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -114,12 +116,23 @@
                     .enumType(NotNullEnforcer.class)
                     .defaultValue(NotNullEnforcer.ERROR)
                     .withDescription(
-                            "The NOT NULL column constraint on a table enforces that "
-                                    + "null values can't be inserted into the table. Flink supports "
-                                    + "'error' (default) and 'drop' enforcement behavior. By default, "
-                                    + "Flink will check values and throw runtime exception when null values writing "
-                                    + "into NOT NULL columns. Users can change the behavior to 'drop' to "
-                                    + "silently drop such records without throwing exception.");
+                            "Determines whether the NOT NULL column constraint on a table enforces that "
+                                    + "null values can't be inserted into the table. By default, "
+                                    + "Flink will check values and throw runtime exception when attempting "
+                                    + "to write null values writing into NOT NULL columns. Users can change "
+                                    + "the behavior to silently drop such records without throwing exception.");
```

Comment:
This entire paragraph just re-states the description of the enum. I think we can simplify this to something like

```suggestion
                            "Determines how Flink enforces NOT NULL column constraints when inserting null values.");
```